client/js/helpers/parse.js

```diff
@@ -79,7 +79,10 @@ function parse(createElement, text, message = undefined, network = undefined) {
 	// arrays of objects containing start and end markers, as well as metadata
 	// depending on what was found (channel or link).
 	const channelPrefixes = network ? network.serverOptions.CHANTYPES : ["#", "&"];
-	const userModes = network ? network.serverOptions.PREFIX : ["!", "@", "%", "+"];
+	const userModes =
+		network && network.serverOptions && network.serverOptions.PREFIX
```

Comment:
network?.serverOptions?.PREFIX should work

Reply:
do we have that compatibility level? That syntax is pretty new.
Will do that then

Reply:
yeah we should support es2020, which supports it